resources.whatwg.org/build/review.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+
+# This script is maintained at
+# https://github.com/whatwg/whatwg.org/tree/master/resources.whatwg.org/build.
+# See README.md there for documentation.
+
+mkdir -p "review-drafts"
+INPUT_FILE=$(find . -maxdepth 1 -name "*.bs" -print -quit)
+REVIEW_DRAFT="review-drafts/$(date +'%Y-%m').bs"
+# The very unusual way of creating a newline here is done to keep macOS happy
+sed 's/^Group: WHATWG$/&'''$'\n'"Date: $(date +'%Y-%m-%d')/g" < "$INPUT_FILE" > "$REVIEW_DRAFT"
```

Comment:
I tried running this locally and got <code>sed: -e expression #1, char 19: unterminated `s' command</code> which is probably about this line. Using sed (GNU sed) 4.4.

Reply:
Hmm, I suspect the various versions of sed are incompatible in their syntaxes. What matters most is that this works on Travis, which we haven't really tried yet. But it does pass shellcheck apparently.

Reply:
I think `'''` also does not work on macOS. `\'` did work.